site/content/en/spec-3.3.md

```diff
@@ -69,7 +69,9 @@ Covered Files are any file which must contain Licensing Information. This is
 equal to all files in a project, with the exception of:
 
 - The License Files stored in the `LICENSES/` directory.
-- `COPYING` and `LICENSE`, with or without file extensions.
+- `COPYING`, `LICENSE`, and `LICENCE`, with or without file extensions. These
+  MAY also contain a dash separator, like `LICENSE-MIT`. These files SHOULD be
```

Comment:
```suggestion
  MAY also contain a dash or dot separator, like `LICENSE-MIT` or `COPYING.GPL`. These files SHOULD be
```

I’m still unsure I like this, seems like a project-specific thing. But if we’re doing it, let’s do it proper and also include the dot. Calling `.GPL.markdown` a file extension would still stretching it quite a bit.

Reply:
implemented